examples/autoscaler-webhook/autoscaler-service.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Define a Service for the autoscaler-webhook
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: autoscaler-webhook-service
+  annotations:
+    service.alpha.kubernetes.io/app-protocols: '{"http":"HTTP"}'
```

Comment:
Curious - what is this for? Not sure its necessary?

Reply:
Actually was copied from      https://github.com/GoogleCloudPlatform/agones/blob/0aa496dbd83db27e85faa444bfd0356f42728fe0/examples/allocator-service/allocator-service.yaml#L7
Not needed here